srcpkgs/python3-cairocffi/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'python3-cairocffi'
+pkgname=python3-cairocffi
+version=1.1.0
+revision=1
+archs=noarch
+wrksrc="cairocffi-${version}"
+build_style=python3-module
+pycompile_module="cairocffi"
+make_install_args="--skip-build"
+hostmakedepends="python3-setuptools python3-cffi python3-xcffib"
+makedepends="cairo-devel"
+# Missing optional dependencies:
```

Comment:
Please clarify is it about missing python3-xcffib in repo or consciously not adding that to depends because it is optional.